doc(templates/extras): `calc` in tactic mode (#245)
leanprover-community/leanprover-community.github.io: templates/extras/calc.md

...    = c + 1 : by rw H2
```

`calc` is also available in tactic mode. You can leave `_`s to create a new goal:
```lean
example (a b c : ℕ) (H1 : a = b + 1) (H2 : b = c) : a = c + 1 :=
begin
  calc a = b + 1 : H1
  ...    = c + 1 : _,
  { rw H2 }
end
```

> In fact, `calc A = B : H ...` in tactic mode functions exactly like a call to `refine (calc A = B : H ...)`.

Reviewer: These ellipses are a little confusing since they're also part of the syntax for `calc`, but I'm not sure of a good solution... maybe this?
```suggestion
In fact, in tactic mode, `calc [...]` functions exactly like a call to `refine (calc [...])`.
```
Should we mention that you can do `exact calc [...]` if you want to override the `refine` behavior? (I'm not sure when you'd want to though!)

Reviewer 2: You can use an ellipsis symbol!
```suggestion
In fact, `calc A = B : H …` in tactic mode functions exactly like a call to `refine (calc A = B : H …)`.
```

Author: I think the ellipsis symbol is still too likely to be confusing. As a third option, perhaps:
```suggestion
In fact, `calc [etc]` in tactic mode functions exactly like a call to `refine (calc [etc])`.
```

Reviewer 2: The square brackets look like they are part of the syntax.

Reviewer 3: I think it all doesn't matter too much. Users likely already understand that part of the syntax. The context is making clear that what's important in this sentence is the beginning of the code example.
